functions/Get-DbaSsisExecutionHistory.ps1

```diff
@@ -65,50 +65,54 @@ function Get-DbaSsisExecutionHistory {
     )
     begin {
         $statuses = @{
-            'created'= 1
-            'running'= 2
-            'cancelled'= 3
-            'failed'= 4
-            'pending'= 5
-            'halted'= 6
-            'succeeded'= 7
-            'stopping'= 8
-            'completed'= 9
-          }
-            if ($Status) {
-                $csv = ($statuses[$Status] -join ',')
-                $statusq = "AND e.[status] in ($csv)"
-            } else {
-                $statusq = ''
-            }
-
-            if ($Project) {
-                $csv = "`"" + ($Project -join'","') + "`""
-                $projectq = "AND e.[project_name] in ($csv)"
-            } else {
-                $projectq = ''
-            }
-
-            if ($Folder) {
-                $csv = "`'" + ($Folder -join"'","'") + "`'"
-                $folderq = "AND e.[folder_name] in ($csv)"
-            } else {
-                $folderq = ''
-            }
-
-            if ($Environment) {
-                $csv = "`'" + ($Environment -join"'","'") + "`'"
-                $environmentq = "AND e.[environment] in ($csv)"
-            } else {
-                $environmentq = ''
-            }
-
-            $q = @"
+            'Created'   = 1
+            'Running'   = 2
+            'Cancelled' = 3
+            'Failed'    = 4
+            'Pending'   = 5
+            'Halted'    = 6
+            'Succeeded' = 7
+            'Stopping'  = 8
+            'Completed' = 9
+        }
+        if ($Status) {
+            $csv = ($statuses[$Status] -join ',')
+            $statusq = "AND e.[Status] in ($csv)"
+        }
+        else {
+            $statusq = ''
+        }
+        
+        if ($Project) {
+            $csv = "`"" + ($Project -join '","') + "`""
+            $projectq = "AND e.[ProjectName] in ($csv)"
+        }
+        else {
+            $projectq = ''
+        }
+        
+        if ($Folder) {
+            $csv = "`'" + ($Folder -join "'", "'") + "`'"
+            $folderq = "AND e.[FolderName] in ($csv)"
+        }
+        else {
+            $folderq = ''
+        }
+        
+        if ($Environment) {
+            $csv = "`'" + ($Environment -join "'", "'") + "`'"
+            $environmentq = "AND e.[Environment] in ($csv)"
+        }
+        else {
+            $environmentq = ''
+        }
+        
+        $sql = "
             WITH
             cteLoglevel as (
                 SELECT
-                    execution_id,
-                    cast(parameter_value AS INT) AS logging_level
```

Comment:
logging_level is SQLy, while LoggingLevel is PowerShelly so I changed the output to look more like PS and less like SQL